crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/storage.rs

```diff
@@ -13,26 +13,24 @@ use starknet_api::state::StorageKey;
 ///
 /// * `blockifier_state`: Blockifier state reader
 /// * `target`: The address of the contract we want to target
-/// * `storage_address`: Beginning of the storage of the variable
-/// * `values`: A vector of values to write starting at `storage_address`
+/// * `storage_address`: Storage address of the felt
```

Comment:
```suggestion
/// * `storage_address`: Address of the storage
```
not the top explanation, but the current one sounds misleading, felts do not have an address

Reply:
I wouldn't agree on this one

Reply:
`Target's storage address`?

Reply:
`Storage address to be set to value`

Reply:
But this function stores only one felt at a certain address, and the address is the address of the felt

Reply:
I guess what you said is correct

Reply:
[Make comment more verbose](https://github.com/foundry-rs/starknet-foundry/pull/1487/commits/dd175a72a555b41a48dabae833eb373bcaa52f30)